src/modules/resource/topo.c

```diff
@@ -304,7 +304,8 @@ static char *topo_get_local_xml (struct resource_ctx *ctx, bool no_restrict)
     }
```

Comment:
"resource" is misspelled in the commit body for `testsuite: add test with resource.norestrict in subinstance`

Reply:
Thanks! Fixed that and force-pushed. Will set MWP.